provider/resource_keycloak_identity_provider_token_exchange_scope_permission.go

```diff
@@ -0,0 +1,292 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"log"
+	"strconv"
+	"strings"
+)
+
+func resourceKeycloakIdentityProviderTokenExchangeScopePermission() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakIdentityProviderTokenExchangeScopePermissionCreate,
+		Read:   resourceKeycloakIdentityProviderTokenExchangeScopePermissionRead,
+		Delete: resourceKeycloakIdentityProviderTokenExchangeScopePermissionDelete,
+		Update: resourceKeycloakIdentityProviderTokenExchangeScopePermissionUpdate,
+		// This resource can be imported using {{realmId}}/{{providerAlias}}. The provider alias is displayed in the URL when editing it from the GUI
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakIdentityProviderTokenExchangeScopePermissionImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"provider_alias": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"policy_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "client",
+				Description: "At the moment only 'client' type is supported",
+			},
+			"clients": {
+				Type:        schema.TypeSet,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				Description: "Ids of the clients for which a policy will be created and set on scope based token exchange permission",
+			},
+			"policy_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				ForceNew:    true,
```

Comment:
why is `ForceNew` needed for a computed attribute?

Reply:
Mistake, I will remove it